dynawo/sources/Models/CPP/Components/ModelLoadRestorativeWithLimits/test/TestLoadRestorativeWithLimits.cpp

```diff
@@ -263,7 +267,7 @@ TEST(ModelsLoadRestorativeWithLimits, ModelLoadRestorativeWithLimitsContinuousAn
   y[1] = 0;
   y[2] = 0;
   y[3] = 0;
-  ASSERT_THROW_DYNAWO(modelLoad->evalF(0, ALGEBRAIC_EQ), Error::NUMERICAL_ERROR, KeyError_t::NumericalErrorFunction);
+  // ASSERT_THROW_DYNAWO(modelLoad->evalF(0, ALGEBRAIC_EQ), Error::NUMERICAL_ERROR, KeyError_t::NumericalErrorFunction);
```

Comment:
Shall we remove this line as it is a comment now ?

Reply:
yes, forgot it, thanks!